includes/class-wcpdf-settings-debug.php

```diff
@@ -348,6 +373,66 @@ public function import_settings( $data ) {
 		}
 	}
 	
+	public function reset_settings( $data ) {
+		extract( $data );
+		
+		if ( empty( $type ) ) {
+			$message = __( 'Reset settings type is empty!', 'woocommerce-pdf-invoices-packing-slips' );
+			wcpdf_log_error( $message );
+			wp_send_json_error( compact( 'message' ) );
+		}
+		
+		$settings_option = '';
+		
+		switch ( $type ) {
+			case 'general':
+				$settings_option = 'wpo_wcpdf_settings_general';
+				break;
+			case 'debug':
+				$settings_option = 'wpo_wcpdf_settings_debug';
+				break;
+			default:
+				$settings_option = apply_filters( 'wpo_wcpdf_reset_settings_option', $settings_option, $type );
```

Comment:
As I installed the Professional extension, I realized that the Pro settings are selectable too:

![image](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/assets/38109855/520d854a-b417-4147-8727-45340cca099d)

But the reset does not work for this settings option yet:

![image](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/assets/38109855/f10c8767-fb20-4389-8500-607046be039a)


Will the Pro settings resetting available in the Professional extension instead, or maybe should we to do something like the following?:

```php
$settings_option = apply_filters( 'wpo_wcpdf_reset_settings_option', "wpo_wcpdf_settings_{$type}", $type );
```

Reply:
Pro and Templates are not ready yet, we need to open new issues there to hook here: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/690defa47ac1713b382554ce56e5bc994140a069/includes/class-wcpdf-settings-debug.php#L395

I have changed the notice message, in case they had a legacy version of the extension without the reset support.

Reply:
https://github.com/wpovernight/woocommerce-pdf-ips-pro/issues/257
https://github.com/wpovernight/woocommerce-pdf-ips-templates/issues/229